src/librustdoc/html/static/css/rustdoc.css

```
@@ -230,6 +230,12 @@ h4.code-header {
 	padding: 0;
 	white-space: pre-wrap;
 }
+.structfield {
+	margin-block-start: .6em;
+	margin-block-end: .6em;
+	margin-inline-start: 0px;
+	margin-inline-end: 0px;
```

Can be done more simply with:

```suggestion
	margin: 0.6em 0;
```